server/tests/trip.test.js

```diff
@@ -111,7 +112,7 @@ describe('Trip test suite', () => {
         done(err);
       });
   });
-  it('5.should not create a new trip if date is invalide', (done) => {
+  it('5.should not create a new trip if date is invalid', (done) => {
```

Comment:
You can remove these starting numbers. No need.

Reply:
removed